bin/tronctl

```diff
@@ -371,6 +384,25 @@ def move(args):
     yield request(urljoin(args.server, "/api/jobs"), data)
 
 
+def get_validated_max_parallel(max_parallel, max_parallel_limit):
+    if max_parallel <= 0:
+        raise SystemExit(f"The flag --max-parallel must be a positive integer.")
+    if max_parallel_limit <= 0:
+        raise SystemExit(f"The flag --max-parallel-limit must be a positive integer.")
+
+    if max_parallel_limit > LIMIT_MAX_PARALLEL_RUNS:
+        print(f"The flag --max-parallel-limit is overwritten to {max_parallel_limit} which is higher than " +
+              f"the suggested limit of {LIMIT_MAX_PARALLEL_RUNS}. If you haven't consulted the Tron team yet, " +
+              f"then please do so now to avoid overloading Tron with too many backfill jobs.")
+
+    if max_parallel > max_parallel_limit:
+        print(f"The flag --max-parallel is set to {max_parallel} but the --max-parallel-limit only allows " +
+              f"{max_parallel_limit}. Overriding concurrency to enforce the limit of {max_parallel_limit}.")
```

Comment:
I would exit here with an error saying the user is past the limit and should consult the Tron team if they need that many parallel runs, instead of proceeding with a `max_parallel` value that the user didn't intend.

Reply:
++

Reply:
we'll also need to figure out what to do if someone does need more - do we just modify a local copy of tronctl to strip out this limiter and run the backfill for them?